### PR TITLE
build: fix package miss files

### PR DESCRIPTION
**Pull request by HerrCai0907**

### CODE REVIEW

**Review by CountBleck — APPROVED** [review]
I'm surprised this is the issue. Is `dist` removed during some part of the publishing process?

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,10 @@ raw/
 cli/index.generated.js
 src/diagnosticMessages.generated.ts
 coverage/
+
+/dist/*.generated.d.ts
+/dist/*.map
+/dist/asc.js
+/dist/assemblyscript.js
+/dist/importmap.json
+/dist/web.js
```
[the remark this answers is not in the document]
Comment (CountBleck) [reply .gitignore:18]: Also, please make sure my suggestion works before merging this PR. For all I know, the leading slash matters.

**Comment by HerrCai0907** [comment]
> I'm surprised this is the issue. Is `dist` removed during some part of the publishing process?

Yes, It is weird that gitignore in dist will influence npm package.

**Comment by CountBleck** [comment]
It appears to be a peculiarity with [the `files` option](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#files) where `.gitignore` at the root has no effect on `files` but `.gitignore` in subdirectories does.

**Comment by JulianDM1995** [comment]
This breaks the npm package

**Comment by CountBleck** [comment]
@JulianDM1995 This hasn't been published yet. In fact, this'll fix the npm package.

**Pull request merged**